fe/fe-core/src/main/java/com/starrocks/connector/PartitionUtil.java

```diff
@@ -793,4 +794,8 @@ public static String getPartitionName(String basePath, String partitionPath) {
     public static String getPathWithSlash(String path) {
         return path.endsWith("/") ? path : path + "/";
     }
+
+    public static String getPartitionSourceName(Schema schema, PartitionField partition) {
+        return schema.findColumnName(partition.sourceId());
```

Comment:
Can you comment here to explain the difference from PartitionField.name() and elaborate on when to use which?

Reply:
Thanks for comments!